src/org/usfirst/frc4904/robot/subsystems/Shooter.java

```diff
@@ -4,28 +4,56 @@
 import java.util.HashMap;
 import java.util.Iterator;
 import java.util.Set;
+import org.usfirst.frc4904.robot.RobotMap;
 import org.usfirst.frc4904.standard.commands.Idle;
-import org.usfirst.frc4904.standard.custom.sensors.CANSensor;
+import org.usfirst.frc4904.standard.custom.sensors.DistanceSensor;
 import edu.wpi.first.wpilibj.command.Subsystem;
 
 public class Shooter extends Subsystem {
 	public final RockNRoller rockNRoller;
 	public final Hood hood;
 	public final Flywheel flywheel;
-	public final CANSensor distanceSensor;
+	public final DistanceSensor towerDistanceSensor;
+	public final DistanceSensor ballLoadSensor;
 	private final HashMap<Double, Double> measuredSpeeds;
 	
-	public Shooter(RockNRoller rockNRoller, Hood hood, Flywheel flywheel, CANSensor distanceSensor) {
+	/**
+	 * Construct a Shooter subsystem with each individual component.
+	 * 
+	 * @param rockNRoller
+	 *        the RockNRoller unit
+	 * @param hood
+	 *        the Hood unit
+	 * @param flywheel
+	 *        the Flywheel unit
+	 * @param towerDistanceSensor
+	 *        the CAN distance sensor
```

Comment:
Change this and the next comment - the distance sensors are implementation agnostic